docs/src/solutions.md

```diff
@@ -128,6 +128,22 @@ end
 # TODO: How to accurately measure the solve time.
 ```
 
+## Accessing MathOptInterface attributes
+
+[MathOptInterface](https://www.juliaopt.org/MathOptInterface.jl/stable/) defines a large
+number of model attributes that can be queried. Examples include
+[`MOI.RelativeGap`](https://www.juliaopt.org/MathOptInterface.jl/stable/apireference/#MathOptInterface.RelativeGap) and
```

Comment:
I think it's better to link to versioned docs. Otherwise we could end up with broken links later.